install/updates/froxlor/0.10/update_0.10.inc.php

```diff
@@ -526,7 +526,9 @@
 
 if (\Froxlor\Froxlor::isDatabaseVersion('201912100')) {
 	showUpdateStep("Adding custom phpfpm configuration field");
-	Database::query("ALTER TABLE `" . TABLE_PANEL_PHPDAEMONS . "` ADD `custom_config` text NOT NULL DEFAULT '' AFTER `limit_extensions`;");
+	Database::query("ALTER TABLE `" . TABLE_PANEL_PHPDAEMONS . "` ADD `custom_config` text AFTER `limit_extensions`;");
+	showUpdateStep("Adding option to disable SSL sessiontickets for older systems");
```

Comment:
need to add a `lastStepStatus(0);` here before the next `showUpdateStep()` call, also, it is recommended to put your changes into a new database-version as testing users might already be on `201912310` and therefore will never receive these updates. If you do that today, just make it `201912311` everywhere